### PR TITLE
Add gps plugins

### PR DESCRIPTION
**Pull request by WenzDaniel**
## What does the code in this PR do / what does it improve?
Removed old syncing plugins and move GPS plugins from cutax to straxen.

### CODE REVIEW

**Comment by WenzDaniel** [comment]
I can fix the reviewdoc things, but not sure if I should wait for the pre-commit thing.

**Comment by WenzDaniel** [comment]
To make tests working we need to updated our test data to a run for which also GPS signals are available: https://github.com/XENONnT/straxen/blob/11cdb5c8ae7e27990a84c269c4c3e7778acae50b/straxen/test_utils.py#L16-L21
This should to be discussed in Team A.

**Comment by WenzDaniel** [comment]
The failing test has something to do with a numerical issue in the event pattern fit which cannot be solved right now.

File: straxen/plugins/gps_syncing/gps_syncing.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    1. Finds the TTL GPS pulses coming into the AM from the gps
+    module and their pairs coming from the module for the
+    correspondent run.
+      2. Corrects the timestamp of all events by linearly interpolating
+    between the previous and next sync pulses.
+
+    """
+
+    __version__ = "0.2.1"
+    depends_on = ("aqmon_hits", "event_info")
```
Comment (jjakob03) [inline straxen/plugins/gps_syncing/gps_syncing.py:24]:
```suggestion
    depends_on = ("aqmon_hits", "event_basics")
```

**Review by jjakob03 — DISMISSED** [review]
Looks good. I did not check mv but nv works :)

**Comment by WenzDaniel** [comment]
Thanks Johanna, the MV plugins were just copied over from cutax. I will try to verify them again.

**Pull request merged**